app/controllers/change_requests_controller.rb

```diff
@@ -180,17 +180,34 @@ def persist_change(change_request)
       phone.update field_change_hash
     elsif change_request.is_a? AddressChangeRequest
       puts "AddressChangeRequest"
+      # action = change_request.field_changes["action"]
+      action = "" # to be removed for end-to-end testing
```

Comment:
The `action` is an enum on the [`ChangeRequest`](https://github.com/ShelterTechSF/askdarcel-api/blob/3ad4773352ddd5d0c089008197a1035d1279733c/app/models/change_request.rb#L5) model, meaning that once it's saved to the database, it is represented as an integer. This means that you should be able to check the enum value directly on the model instance, and not the `field_change_hash`, using either the name of the enum property or the special boolean methods for each enum value like so:

```rb
change_request.status  # Gives you a string holding "insert", "edit", or "remove"
change_request.remove? # Returns true if and only if the status is set to "remove"
```

See https://api.rubyonrails.org/v5.2.3/classes/ActiveRecord/Enum.html for some docs on how Rails enums work.